src/visualizations/config/index.js

```diff
@@ -42,6 +36,13 @@ export default function({
     }
 
     this.getConfig = () => {
+        const DEFAULT_EXTRA_OPTIONS = {
+            colors: theme1,
+            noData: {
+                text: i18n.t('No data'),
```

Comment:
Is the text itself supposed to be changed? `No data to display` -> `No data`

Reply:
Yes, that was in discussion with Jan.